Assets/SEE/Game/SceneManipulation/GameNodeMarker.cs

```diff
@@ -0,0 +1,54 @@
+using SEE.Game.City;
+using SEE.GO;
+using SEE.Utils;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SEE.Game.Scenemanipulation
```

Comment:
`Scenemanipulation` must be `SceneManipulation`. Note that C# is case-sensitive.